karma.mc.config.js

```diff
@@ -41,7 +42,12 @@ module.exports = function(config) {
             exclude: [/node_modules/],
             use: [{
               loader: "babel-loader",
-              options: {plugins: [["jsm-to-esmodules", {basePath: PATHS.resourcePathRegEx, replace: true}]]}
+              options: {
+                plugins: [
+                  ["jsm-to-esmodules", {basePath: PATHS.resourcePathRegEx, replace: true}],
+                  "rewire"
```

Comment:
It would be nice if each of the plugins had a short comment explaining what it was doing and why...